File: src/com/intellij/advancedExpressionFolding/expression/math/trig/Atan2.java
package com.intellij.advancedExpressionFolding.expression.math.trig;

import com.intellij.advancedExpressionFolding.expression.Expression;
import com.intellij.advancedExpressionFolding.expression.Function;
import com.intellij.advancedExpressionFolding.expression.math.ArithmeticExpression;
import com.intellij.openapi.util.TextRange;
import com.intellij.psi.PsiElement;

import java.util.List;

public class Atan2 extends Function implements ArithmeticExpression {
    public Atan2(PsiElement element, TextRange textRange, List<Expression> operands) {
        super(element, textRange, "atan2", operands);
    }
}

File: src/com/intellij/advancedExpressionFolding/expression/math/trig/Atan2.kt
package com.intellij.advancedExpressionFolding.expression.math.trig

import com.intellij.advancedExpressionFolding.expression.Expression
import com.intellij.advancedExpressionFolding.expression.Function
import com.intellij.advancedExpressionFolding.expression.math.ArithmeticExpression
import com.intellij.openapi.util.TextRange
import com.intellij.psi.PsiElement


class Atan2(
    element: PsiElement,
    textRange: TextRange,
    operands: List<Expression>
) : Function(element, textRange, "atan2", operands), ArithmeticExpression

